sql/moz-fx-data-shared-prod/telemetry_derived/experiment_enrollment_other_events_overall_v1/view.sql

```diff
@@ -6,55 +6,25 @@ WITH pivot AS (
     window_start,
     experiment,
     branch,
-    "graduated" AS event,
-    graduate_count AS count
-  FROM
-    `moz-fx-data-shared-prod.telemetry_derived.experiment_enrollment_aggregates_live_v1`
-  UNION ALL
-  SELECT
-    window_start,
-    experiment,
-    branch,
-    "enroll_failed" AS event,
-    enroll_failed_count AS count
-  FROM
-    `moz-fx-data-shared-prod.telemetry_derived.experiment_enrollment_aggregates_live_v1`
-  UNION ALL
-  SELECT
-    window_start,
-    experiment,
-    branch,
-    "unenroll_failed" AS event,
-    unenroll_failed_count AS count
-  FROM
-    `moz-fx-data-shared-prod.telemetry_derived.experiment_enrollment_aggregates_live_v1`
-  UNION ALL
-  SELECT
-    window_start,
-    experiment,
-    branch,
-    "updated" AS event,
-    update_count AS count
-  FROM
-    `moz-fx-data-shared-prod.telemetry_derived.experiment_enrollment_aggregates_live_v1`
-  UNION ALL
-  SELECT
-    window_start,
-    experiment,
-    branch,
-    "update_failed" AS event,
-    update_failed_count AS count
+    [
+      STRUCT("graduated" AS event, graduate_count AS count),
+      STRUCT("enroll_failed" AS event, enroll_failed_count AS count),
+      STRUCT("unenroll_failed" AS event, unenroll_failed_count AS count),
+      STRUCT("updated" AS event, update_count AS count),
+      STRUCT("update_failed" AS event, update_failed_count AS count)
+    ] AS events
   FROM
     `moz-fx-data-shared-prod.telemetry_derived.experiment_enrollment_aggregates_live_v1`
 )
 SELECT
   window_start AS `time`,
   experiment,
   branch,
-  event,
-  SUM(`count`) AS value
+  event.event AS event,
+  SUM(event.count) AS value
 FROM
-  pivot
+  pivot,
+  UNNEST(events) AS event
```

Comment:
I'd prefer an explicit CROSS JOIN here, and calling this `e` rather than `event` so that we can have `e.event` rather than `event.event`